lib/LaTeXML/Package/csquotes.sty.ltxml

```diff
@@ -27,11 +27,11 @@ InputDefinitions('csquotes', type => 'sty', noltxml => 1);
 ###########################################################################
 # Compatibility fixes
 
```

Comment:
Taking the comments out from TeX into Perl is just a minor cosmetic change, since the performance implications were irking me while inspecting the binding.